packages/nextjs/utils/scaffold-eth/useNetworkColor.ts

```diff
@@ -1,10 +1,9 @@
-import { getNetworkDetailsByChainId } from "./networks";
+import { getTargetNetwork } from "./networks";
 import { useDarkMode } from "usehooks-ts";
 
-export function useNetworkColor(chainId?: number) {
+export function useNetworkColor() {
```

Comment:
we can mention a short JsDoc comment 

maybe - "gives you the color of the current configured/target network"

Reply:
Thanks, @technophile-04 ! Added JsDoc. But chainId is still removed since we don't need it for now and it intersects with `getTargetNetwork` change https://github.com/scaffold-eth/se-2/pull/187. Let's add it when we need it.

Reply:
Added the comment + moved the "util" to the "hook" folder (since I think it should be there - it's using a hook inside it)

> also was wondering if should we accept chainId ?

I think this could be useful at some point. But let's add it if we end needing it.

Reply:
Sorry, didn't see your comment @rin-st 

fixing it now

Reply:
Sorry for the push force :D